src/grpc/router_ics_eui_worker.erl

```diff
@@ -185,30 +184,35 @@ handle_cast({?RECONCILE_START, Options}, #state{conn_backoff = Backoff0} = State
             {noreply, State#state{conn_backoff = Backoff2}}
     end;
 handle_cast(
-    {?RECONCILE_END, #{forward_pid := Pid, commit := false} = Options, EUIPairs},
+    {?RECONCILE_END, #{error := Error} = Options, _EUIPairs},
```

Comment:
I restructured the `reconcile_end` callbacks.

The list handler module will add an `error` key to `Options` if it gets a trailer other than `<<"0">>`, then the stream will get an `eos` and forward back to this module.

First we handle Errors:
- forward to the caller (if there's a pid)
- maybe schedule a reconnect (if there is not a pid)

Then non-commits:
- compare to local db of records
- forward to caller (if you can get here without a forward pid, congrats you made some logs)

And lastly, commits:
- compare to local db of records
- forward to caller
- maybe reschedule if the update fails (only if there's no pid)